--- a/xinitrc.blackbox	2021-08-05 00:16:52.000000000 -0500
+++ b/xinitrc.blackbox	2026-07-07 17:22:24.988703805 -0500
@@ -6,6 +6,13 @@
 sysresources=/etc/X11/xinit/.Xresources
 sysmodmap=/etc/X11/xinit/.Xmodmap
 
+export XMODIFIERS="@im=none"
+export GTK_IM_MODULE="xim"
+export QT_IM_MODULE="xim"
+
+# For GNOME.
+# export CLUTTER_IM_MODULE="xim"
+
 # merge in defaults and keymaps
 
 if [ -f $sysresources ]; then
@@ -24,6 +31,12 @@
     xmodmap $usermodmap
 fi
 
+setxkbmap -option compose:f13
+
+if type xcape >/dev/null 2>&1; then
+    xcape -e 'Alt_L=F13'
+fi
+
 # Start the window manager:
 if [ -z "$DESKTOP_SESSION" -a -x /usr/bin/ck-launch-session ]; then
   exec ck-launch-session /usr/bin/startblackbox
